cypress/e2e/shared/explorer.test.ts

```diff
@@ -1526,4 +1526,90 @@ describe('DataExplorer', () => {
       // TODO: add filter values based on dropdown selection in key / value
     })
   })
+
+  describe('simple table interactions', () => {
+    const simpleSmall = 'simple-small'
+    const simpleLarge = 'simple-large'
+    beforeEach(() => {
+      cy.window().then(win => {
+        win.influx.set('simpleTable', true)
+      })
+
+      cy.get('@org').then(({id: orgID}: Organization) => {
+        cy.getByTestID('tree-nav')
+        cy.createBucket(orgID, name, simpleLarge)
+        cy.writeData(lines(300), simpleLarge)
+        cy.createBucket(orgID, name, simpleSmall)
+        cy.writeData(lines(30), simpleSmall)
+        cy.reload()
+      })
+    })
+
+    it('should render correctly after switching from a dataset with more pages to one with fewer', () => {
+      cy.getByTestID('query-builder').should('exist')
+
+      // show raw data view of data with 100 pages
+      cy.getByTestID(`selector-list ${simpleLarge}`).should('be.visible')
+      cy.getByTestID(`selector-list ${simpleLarge}`).click()
+
+      cy.getByTestID('selector-list m').should('be.visible')
+      cy.getByTestID('selector-list m').clickAttached()
+
+      cy.getByTestID('selector-list v').should('be.visible')
+      cy.getByTestID('selector-list v').clickAttached()
+
+      cy.getByTestID('selector-list tv1').clickAttached()
+
+      cy.getByTestID('time-machine-submit-button').click()
+
+      cy.getByTestID('raw-data--toggle').click()
+      cy.getByTestID('table').should('exist')
+
+      // click last page
+      cy.getByTestID('pagination-item')
+        .last()
+        .should('be.visible')
+      cy.getByTestID('pagination-item')
+        .last()
+        .click()
+      // verify correct number of pages
+      cy.getByTestID('pagination-item')
+        .last()
+        .contains('100')
+
+      // show raw data view of data with 10 pages
+      cy.getByTestID(`selector-list ${simpleSmall}`).should('be.visible')
+      cy.getByTestID(`selector-list ${simpleSmall}`).click()
+
+      cy.getByTestID('selector-list m').should('be.visible')
+      cy.getByTestID('selector-list m').clickAttached()
+
+      cy.getByTestID('selector-list v').should('be.visible')
+      cy.getByTestID('selector-list v').clickAttached()
+
+      cy.getByTestID('selector-list tv1').clickAttached()
+
+      cy.getByTestID('time-machine-submit-button').click()
+
+      // verify table still exists
+      cy.getByTestID('table').should('exist')
+      // TODO: uncomment when this bug in clockface is fixed: https://github.com/influxdata/clockface/pull/635
```

Comment:
should we write an issue to uncomment this with the next clockface bump?

Reply:
added clockface bump to this PR and uncommented this!